js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_de.properties

```diff
@@ -177,3 +202,438 @@ category=Kategorie
+algorithm=Algorithmus
+use=Verwendung
+provider=Anbieter
+validTo=Gültig bis
+publicKeys=Öffentlicher Schlüssel
+eventListeners=Ereignis-Listener
+eventListenersHelpTextHelp=Ereignis-Listener die Ereignisse für den Realm behandeln.
+userEventsSettings=Benutzerereignisse
+adminEventsSettings=Administrationsereignisse
+saveEvents=Ereignisse speichern
+save-user-eventsHelp=Wenn diese Option aktiviert ist, werden die Benutzerereignisse in der Datenbank gespeichert, wodurch die Ereignisse für die Administrations- und Benutzerkontokonsolen verfügbar werden.
+revert=Zurücksetzen
+clearUserEvents=Benutzerereignisse löschen
+user-clearEventsHelp=Löscht alle Benutzerereignisse in der Datenbank.
+refresh=Aktualisieren
+addSavedTypes=Gespeicherte Ergeignis-Typen hinzufügen
+searchEventType=Gespeicherte Ereignisart suchen
+eventType=Art des gespeicherten Ereignisses
+expiration=Ablaufdatum
+expirationHelp=Legt das Ablaufdatum für Ereignisse fest. Abgelaufene Ereignisse werden in regelmäßigen Abständen aus der Datenbank gelöscht.
+on=An
+off=Aus
+clearAdminEvents=Admin-Ereignisse löschen
+admin-clearEventsHelp=Löscht alle Verwaltungsereignisse in der Datenbank.
+includeRepresentation=Repräsentation speichern
+includeRepresentationHelp=JSON-Darstellung für Erstellungs- und Aktualisierungsanfragen speichern.
+internationalization=Internationalisierung
+internationalizationHelp=Wenn diese Option aktiviert ist, können Sie wählen, welche Gebietsschemata Sie für diesen Bereich unterstützen und welches Gebietsschema standardmäßig verwendet werden soll.
+locales=Gebietsschema
+defaultLocale=Standard Gebietsschema
+realmOverrides=Realm Überschreibungen
+realmOverridesDescription=Mit Realm-Überschreibungen können Sie Übersetzungen angeben, die für den gesamten Realm gelten sollen. Diese Übersetzungen haben Vorrang vor allen Übersetzungen, die in einem Thema angegeben sind.
+searchForTranslation=Nach Übersetzung suchen
+addTranslation=Übersetzung hinzufügen
+noTranslations=Keine eigenen Übersetzungen vorhanden
+noTranslationsInstructions=Fügen Sie eigene Übersetzungen hinzu.
+effectiveMessageBundles=Effektive Message-Bundles
+effectiveMessageBundlesHelp=Sie können nach effektiven Message-Bundles auf der Grundlage von Thema, Funktionen, Sprache und freiem Text suchen.
+effectiveMessageBundlesDescription=Ein effektives Message-Bundle ist der Satz von Übersetzungen für eine bestimmte Sprache, ein bestimmtes Thema und einen bestimmten Thementyp. Es berücksichtigt auch alle Überschreibungen des Bereichs, die Vorrang haben.
+searchForEffectiveMessageBundles=Nach Message-Bundle suchen
+theme=Thema
+themeType=Thema Typ
+language=Sprache
+hasWords=Mit Worten
+partialImport=Partieller Import
+partialExport=Partieller Export
+securityDefences=Sicherheitsmaßnahmen
+bruteForceDetection=Brute-Force-Erkennung
+bruteForceMode=Brute-Force Modus
+bruteForceMode.Disabled=Deaktiviert
+bruteForceMode.PermanentLockout=Dauerhaft sperren
+bruteForceMode.TemporaryLockout=Vorübergehende Sperre
```

Comment:
"Vorübergehend / temporär sperren", consistent with the permanent lockout translation?

Reply:
I go with:
```
bruteForceMode.PermanentLockout=Dauerhaft sperren
bruteForceMode.TemporaryLockout=Temporär sperren
bruteForceMode.PermanentAfterTemporaryLockout=Dauerhaft sperren nach temporärer Sperre
```